### PR TITLE
chore(python-buildpack): release of Python 3.12.2 and 3.11.8

### PR DESCRIPTION
**Pull request by Frzk**
Related to https://github.com/Scalingo/python-buildpack/issues/93

### CODE REVIEW

File: src/_posts/languages/python/2000-01-01-start.md [hunk 1]
```diff
@@ -14,16 +14,33 @@ To ensure our deployment system considers your application as a Python
...
+{% note %}
+Even though we still support them, we strongly advise against using deprecated
+versions of Python, such as Python `3.7.x`. If you are still using one of these
+deprecated versions, please consider migrating to a more recent version.
+{% endnote %}
+
+### Selecting a Version
+
+The default Python version on both `scalingo-20` and `scalingo-22` is the
+latest `3.12` version.\
```
Comment (EtienneM) [inline src/_posts/languages/python/2000-01-01-start.md:40]: question What is this backslash about? :thinking:

Comment (Frzk) [reply src/_posts/languages/python/2000-01-01-start.md:40]: It creates a `<br />` a line-feed without starting a new paragraph.

Comment (EtienneM) [reply src/_posts/languages/python/2000-01-01-start.md:40]: I'm not sure we want that kind of thing but OK

**Review by EtienneM — APPROVED** [review]

File: src/_posts/languages/python/2000-01-01-start.md (hunk 1 above)
Comment (Frzk) [reply src/_posts/languages/python/2000-01-01-start.md:40]: I can remove it if you want, I don't care, really :smiley_cat:

**Pull request merged**